magefiles/test.go

```diff
@@ -309,3 +312,33 @@ func (t Test) PatchAddonOperatorCSVWebhook(ctx context.Context) error {
 
 	return nil
 }
+
+func injectENVcsv(csvInput operatorsv1alpha1.ClusterServiceVersion, key string, value string) (csvOut operatorsv1alpha1.ClusterServiceVersion) {
```

Comment:
Minor naming tweaks, plus `csvOut` wasn't referenced anywhere so we can drop variable naming.
```suggestion
func injectEnv(csv operatorsv1alpha1.ClusterServiceVersion, key string, value string)  operatorsv1alpha1.ClusterServiceVersion {
```